Week2/MohammadDarwesh/mohammaddarwesh.js

```diff
@@ -0,0 +1,180 @@
+// for the first task i will do it with many ways just for practising.
+console.log ("Halo, dunia! ,Ciao, mondo! ,Hola, mundo!");
+
+let indonesian = "Halo, dunia!";
+let italian = "Ciao, mondo!";
+let spanish = "Hola, mundo!";
+console.log(indonesian ,italian, spanish );
+
+let helloWorld = [`${indonesian}` ,`${italian}` ,`${spanish}`];
+console.log(helloWorld);
+console.log(helloWorld[0]);
+console.log(helloWorld[1]);
+console.log(helloWorld[2]);
+
+let helloWorld2 = ["Halo, dunia!" ,"Ciao, mondo!" ,"Hola, mundo!"];
+console.log(helloWorld2);
+console.log(helloWorld2[0]);
+console.log(helloWorld2[1]);
+console.log(helloWorld2[2]);
+console.log(helloWorld2.join("-"));
+console.log(helloWorld2.reverse());
+
+let helloWorld3 = {
+    indonesian: "Halo, dunia!",
+    italian: "Ciao, mondo!",
+    spanish: "Hola, mundo!"
+   };
+console.log(helloWorld3);
+console.log(helloWorld3.indonesian);
+console.log(helloWorld3.italian);
+console.log(helloWorld3.spanish);
+// end of task 1.
+
+
+// task 2:
+console.log("I'm awesome");
+
+// task 3:
+let x;
+console.log("the value of my variable x will be: 32");
+console.log(x);
+x = 32;
+console.log("x will be :32 ");
+console.log(x);
+
+// task 4:
+let y = "Hello, Yash"
+console.log("the value of my string will be: Hello, Yash ");
+console.log(y);
+y = "Hiiii, Yash"
+console.log("the value of my string will be: Hiiii, Yash ");
+console.log(y);
+
+// task 5:
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+console.log(a > z , z > a );
+if (z > a) {
+    let m = z;
+    console.log(m);
+}
+else{ 
+    m = a;
+    console.log(m);}
+
+
+// task 6:
+let animals;
+console.log("the value of my array will be: cat, dog, hamster");
+console.log(animals);
+let myFavoriteAnimals = ["cat", "dog", "hamster"];
+console.log(myFavoriteAnimals);
+myFavoriteAnimals[3] = "baby pig"
```

Comment:
Even though this works just fine, if you are required to add an element to an array possibly at the end of current array, you should try to use the `push` method we discussed during our class.

That's because although we know the current size of the array, we can't be really sure that it would always contain 3 elements. What if it were supposed to contain 5 elements or we didn't know what the length of the array is. In such case, doing `myFavouriteAnimals[3] = "baby pig"` wouldn't work.

`myFavouriteAnimals.push("baby pig");` would be safer in such a case, right?

Reply:
same as Math.max .....i think also will be better if the if the question more specific .